Docs: update manual grading display (#5890)
PrairieLearn/PrairieLearn: docs/manualGrading.md

If you prefer to use points rather than a percentage score, rename the `score_perc` column in the CSV file to `points`.

> If a feedback column is provided, it will be shown by default to the student above the submission panel, as seen in the image below. This feedback accept Markdown formatting.

Reviewer: Nit:
```suggestion
If a `feedback` column is provided, it will be shown by default to the student above the submission panel, as seen in the image below. The feedback can utilize Markdown formatting.
```
